tests/integration/awslambda/test_lambda_integration.py

```diff
@@ -420,6 +420,139 @@ def verify_failure_received():
         finally:
             lambda_client.delete_event_source_mapping(UUID=event_source_mapping_uuid)
 
+    @pytest.mark.parametrize(
+        "item_to_put1, item_to_put2, filter, calls",
+        [
+            # Test with no filter, and two times same entry (aka MODIFY)
+            (
+                {"id": {"S": "test123"}},
+                None,
+                None,
```

Comment:
this will set `"FilterCriteria": {"Filters": [{"Pattern": "null"}]}}` and will throw an error when running the test against AWS:
```
An error occurred (InvalidParameterValueException) when calling the CreateEventSourceMapping operation: Invalid filter pattern definition.
```